servers/src/main/java/tachyon/worker/block/allocator/Allocator.java

```diff
@@ -56,14 +57,12 @@ public static Allocator createAllocator(TachyonConf conf, BlockMetadataManagerVi
    * or {@link BlockStoreLocation#anyTier()} or {@link BlockStoreLocation#anyDirInTier(int)}.
    * The view is generated by a {@link tachyon.worker.block.BlockStore}.
    *
-   * @param userId the ID of user to apply for the block allocation
-   * @param blockId the ID of the block
    * @param blockSize the size of block in bytes
    * @param location the location in block store
    * @param view of the metadata manager
-   * @return a temp block meta if success, null otherwise
+   * @return a StorageDirView in which to create the temp block meta if success, null otherwise
    * @throws IllegalArgumentException if block location is invalid
    */
-  TempBlockMeta allocateBlockWithView(long userId, long blockId, long blockSize,
-      BlockStoreLocation location, BlockMetadataManagerView view);
+  StorageDirView allocateBlockWithView(long blockSize, BlockStoreLocation location,
```

Comment:
Right, `userId` is not used now with the change. Do you want to keep it and reserve it for the future use?